### PR TITLE
Sync policies with security-monitoring repo

### PR DESCRIPTION
**Pull request by lb5tr**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: runtime/default.policy [hunk 1]
```diff
@@ -464,6 +464,9 @@ rules:
       && open.flags & OPEN_WRITE_FLAGS > 0
       && process.file.path not in SYSTEM_PACKAGE_MANAGERS
       && process.ancestors.file.path not in SYSTEM_PACKAGE_MANAGERS
+  - id: selinux_disable_enforcement
```
Comment (paulcacheux) [inline runtime/default.policy:467]: Are we sure we want this rule ? I see that this rule was added/removed multiple times in the past weeks

Comment (lb5tr) [reply runtime/default.policy:467]: I'm not sure tbh, it's what the generation script created. Not sure who the owner of this is but if this is not needed, then it should be removed from internal repo too.

**Review by paulcacheux — APPROVED** [review]

File: runtime/default.policy (hunk 1 above)
Comment (lebauce) [reply runtime/default.policy:467]: I think part of the problem is that this SElinux rule was only present in security-agent-policies and not in security-monitoring. So when we did a sync, the one in security-agent-policies was removed.
So I think it should be in security-agent-policies. If it causes any issue, we can remove in security-monitoring and sync again

**Review by lebauce — APPROVED** [review]

File: runtime/default.policy (hunk 1 above)
Comment (paulcacheux) [reply runtime/default.policy:467]: Ok let's merge then

**Pull request merged**